server/app/controllers/lists.js

```diff
@@ -0,0 +1,134 @@
+const ErrorResponse = require('../utils/errorResponse');
+const asyncHandler = require('../middleware/async');
+const List = require('../models/List');
+
+/**
+ * @api {get} api/v1/users/:userId/lists
+ * @apiName getLists
+ * @apiGroup lists
+ *
+ * @apiDescription Get all of the lists for a user
+ */
+// eslint-disable-next-line no-unused-vars
+const getLists = asyncHandler(async (req, res, next) => {
+  const { userId } = req.params;
+  const lists = await List.find({ user: userId });
+  if (!lists) {
+    return next(
+      new ErrorResponse(
+        `No lists with the user id of ${req.params.userId}`,
```

Comment:
You've destructured the `userId` from `req.params`, so `req.params.userId` can be replaced with `userId`